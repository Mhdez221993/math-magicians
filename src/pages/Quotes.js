import React from 'react';

const Quotes = () => (
  <div className="description">
    <h2>Math Magicians</h2>
    <p>
      Mathematics includes the study of such topics as quantity, structure, space,
      and change. It has no generally accepted definition.
      Mathematicians seek and use patterns to formulate new conjectures;
      they resolve the truth or falsity of such by mathematical proof.
    </p>
  </div>
);
export default Quotes;
