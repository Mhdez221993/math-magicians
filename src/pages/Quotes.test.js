import { render } from '@testing-library/react';
import React from 'react';
import Quotes from './Quotes';

describe('Quotes', () => {
  it('renders correctly', () => {
    const { getByText } = render(<Quotes />);

    expect(getByText('Math Magicians')).toBeInTheDocument();
  });
});
