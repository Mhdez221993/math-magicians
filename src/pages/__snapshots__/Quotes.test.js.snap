// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test quotes inner content and correct rendering of the Quotes page Quotes page is rendered correctly 1`] = `
<div
  className="description"
>
  <h2>
    Math Magicians
  </h2>
  <p>
    Mathematics includes the study of such topics as quantity, structure, space, and change. It has no generally accepted definition. Mathematicians seek and use patterns to formulate new conjectures; they resolve the truth or falsity of such by mathematical proof.
  </p>
</div>
`;
